pkg/scheduler/framework/plugins/noderesources/balanced_allocation_test.go

```diff
@@ -291,16 +297,90 @@ func TestNodeResourcesBalancedAllocation(t *testing.T) {
 			args:        config.NodeResourcesBalancedAllocationArgs{Resources: defaultResourceBalancedAllocationSet},
 			runPreScore: false,
 		},
+		{
+			// Node1 scores on 0-MaxNodeScore scale
+			// CPU Fraction: 3000 / 35000 = 0.8571
+			// Memory Fraction: 0 / 40000 = 0
+			// DRA Fraction: 1 / 8 = 0.125
+			// Fraction mean: (0.8571 + 0 + 0.125) / 3 = 0.3274
+			// Node1 std: sqrt(((0.8571 - 0.3274)**2 + (0- 0.3274)**2 + (0.125 - 0.3274)**2) / 3) = 0.378
+			// Node1 Score: (1 - 0.378)*MaxNodeScore = 62
+			// Node2 scores on 0-MaxNodeScore scale
+			// CPU Fraction: 3000 / 35000 = 0.8571
+			// Memory Fraction: 5000 / 40000 = 0.125
+			// Node2 std: (0.8571 - 0.125) / 2 = 0.36605
+			// Node2 Score: (1 - 0.36605)*MaxNodeScore = 63
+			pod:          st.MakePod().Req(map[v1.ResourceName]string{extendedResourceDRA: "1"}).Obj(),
+			nodes:        []*v1.Node{makeNode("node1", 3500, 40000, nil), makeNode("node2", 3500, 40000, nil)},
+			expectedList: []fwk.NodeScore{{Name: "node1", Score: 62}, {Name: "node2", Score: 63}},
+			name:         "include DRA resource on a node for balanced resource allocation",
```

Comment:
nit: add 'if PreScore not called' to the name to make it clear this test case if for runPreScore: true